src/geo/tiles.ts

```diff
@@ -229,20 +189,15 @@ export function featureToBboxToTilesCount(feature: Feature<Polygon | MultiPolygo
  * based on 2:1 tile scheme
  * use the bboxToTileRange method and provide sanitized bbox coverage of tiles
  * @param fc
```

Comment:
add explanation on "maxResolutionDeg" and "minResolutionDeg"

Reply:
done